exercise-core/src/main/java/com/examples/attsw/exercise/core/model/Employee.java

```diff
@@ -2,14 +2,25 @@
 
 public class Employee {
 
+	private String name;
 	private String id;
 
-	public Employee(String id) {
+	public Employee(String name, String id) {
```

Comment:
update constructor according to the order of the field (see above); use Eclipse refactoring.